### PR TITLE
Feature/helloworld controller

### PR DESCRIPTION
**Pull request by tyambulatov**
Added hello controller. Spring MVC configured using Java confg.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,31 +6,76 @@
...
     <groupId>ru.timur.learning</groupId>
     <artifactId>battleship</artifactId>
+    <packaging>war</packaging>
     <version>1.0-SNAPSHOT</version>
 
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.version>3.10.1</maven.compiler.version>
+        <maven.war.plugin>3.3.2</maven.war.plugin>
```
Comment (NurmukhametovAlexey) [inline pom.xml]: Let's use consistent naming for version properties `maven.war-plugin.version` and `maven.compiler-plugin.version`

File: src/main/java/ru/timur/learning/controller/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@Controller
+public class HelloController {
+
+    @GetMapping("/hello")
+    public String printWelcome(ModelMap model) {
+
+        model.addAttribute("message", "Spring 3 MVC Hello World");
+        return "hello";
+
```
Comment (NurmukhametovAlexey) [inline src/main/java/ru/timur/learning/controller/HelloController.java]: Lets remove extra blank lines 12 and 15

**Review by NurmukhametovAlexey — DISMISSED** [review]
Nice, just a few style issues

File: src/main/java/ru/timur/learning/controller/HelloController.java (hunk 2 above)
Comment (tyambulatov) [reply src/main/java/ru/timur/learning/controller/HelloController.java]: Resolved by commit 2b9f486

File: pom.xml (hunk 1 above)
Comment (tyambulatov) [reply pom.xml]: Resolved by commit 2b9f486

**Review by NurmukhametovAlexey — APPROVED** [review]

**Pull request merged**